cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -93,80 +95,88 @@ def test_html5_vm_console(appliance, provider, configure_websocket, vm_obj,
         # Get the login screen image, and make sure it is a jpeg file:
         screen = vm_console.get_screen()
         assert imghdr.what('', screen) == 'jpeg'
-
-        assert vm_console.wait_for_text(text_to_find="login:", timeout=200), ("VM Console"
-            " didn't prompt for Login")
-
-        # Enter Username:
-        vm_console.send_keys(console_vm_username)
-
-        assert vm_console.wait_for_text(text_to_find="Password", timeout=200), ("VM Console"
-            " didn't prompt for Password")
-        # Enter Password:
-        vm_console.send_keys("{}\n".format(console_vm_password))
-
-        time.sleep(5)  # wait for login to complete
-
-        # This regex can find if there is a word 'login','password','incorrect' present in
-        # text, irrespective of its case
-        regex_for_login_password = re.compile(r'\blogin\b | \bpassword\b| \bincorrect\b',
-         flags=re.I | re.X)
-
-        def _validate_login():
-            """
-            Try to read what is on present on the last line in console.
-
-            If it is word 'login', enter username, if 'password' enter password, in order
-            to make the login successful
-            """
-            if vm_console.find_text_on_screen(text_to_find='login', current_line=True):
-                vm_console.send_keys(console_vm_username)
-
-            if vm_console.find_text_on_screen(text_to_find='Password', current_line=True):
-                vm_console.send_keys("{}\n".format(console_vm_password))
-            # if the login attempt failed for some reason (happens with RHOS-cirros),
-            # last line of the console will contain one of the following words:
-            # [login, password, incorrect]
-            # if so, regex_for_login_password will find it and result will not be []
-            # .split('\n')[-1] splits the console text on '\n' & picks last item of resulting list
-            result = regex_for_login_password.findall(vm_console.get_screen_text().split('\n')[-1])
-            return result == []
-
-        # if _validate_login() returns True, it means we did not find any of words
-        # [login, password, incorrect] on last line of console text, which implies login success
-        wait_for(func=_validate_login, timeout=300, delay=5)
-
-        logger.info("Wait to get the '$' prompt")
-        if provider.one_of(VMwareProvider):
-            vm_console.wait_for_text(text_to_find=provider.data.templates.get('console_template')
-                            ['prompt_text'], timeout=200)
+        if provider.one_of(OpenStackProvider):
+            # OpenStack VM Console has selenium issue where selenium fails to focus on
+            # canvas and cannot send keys or interact, hence only basic check that VM Console
+            # does have some text displayed in it is all we will do.
+            assert vm_console.get_screen_text() != '', "Console Screen is blank"
```

Comment:
`get_screen_text()` does return non-empty string here which means console was loaded and displayed.